contracts/CometRewards.sol

```diff
@@ -32,13 +32,13 @@ contract CometRewards {
     /// @notice Rewards claimed per Comet instance and user account
     mapping(address => mapping(address => uint)) public rewardsClaimed;
 
-
     /// @dev The scale for factors
     uint256 internal constant FACTOR_SCALE = 1e18;
 
     /** Custom events **/
 
     event GovernorTransferred(address indexed oldGovernor, address indexed newGovernor);
+    event RewardsClaimedSet(address indexed user, address indexed comet, uint256 amount);
```

Comment:
Seems a little inconsistent to emit `comet` for this one but not `RewardsClaimed`

Reply:
Agreed. We could add `comet` to `RewardClaimed`, but unfortunately it won't be indexed because we already have three other indexed fields.